src/Utilities/TrackedDependencies/FlatTrackingData.cs

```diff
@@ -16,6 +16,9 @@
 
 namespace Microsoft.Build.Utilities
 {
+    /// <summary>
+    /// 
```

Comment:
Class used to store and interrogate inputs and outputs recorded by tracking operations.